test/sql/decimal_functions_sql_test.cpp

```diff
@@ -0,0 +1,95 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// decimal_functions_sql_test.cpp
+//
+// Identification: test/sql/decimal_functions_sql_test.cpp
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+
+#include "sql/testing_sql_util.h"
+#include "catalog/catalog.h"
+#include "common/harness.h"
+#include "concurrency/transaction_manager_factory.h"
+
+namespace peloton {
+namespace test {
+
+class DecimalFunctionsSQLTest : public PelotonTest {};
+
+  TEST_F(DecimalFunctionsSQLTest, CeilTest) {
+    auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+    auto txn = txn_manager.BeginTransaction();
+    catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, txn);
+    catalog::Catalog::GetInstance()->Bootstrap();
+    txn_manager.CommitTransaction(txn);
+    // Create a t
+    txn = txn_manager.BeginTransaction();
```

Comment:
This BeginTransaction needs a corresponding EndTransaction else there's a memory leak - refer to my pull request. I had made the same mistake.